ldap/servers/slapd/connection.c

```diff
@@ -300,7 +306,8 @@ connection_reset(Connection *conn, int ns, PRNetAddr *from, int fromLen __attrib
             }
```

Comment:
'from' is craft by the client so it can contain wrong data. I wonder if in case of proxy_connection we should fail if "from->raw.family == PR_AF_LOCAL".

Reply:
Unless mistaken checking that incoming address is trusted should take care of that case without having to explicitly test the familly.